app/models/street/StreetEdgeTable.scala

```diff
@@ -362,10 +362,10 @@ object StreetEdgeTable {
     edges
   }
 
-  def selectAuditedStreetsIntersecting(minLat: Double, minLng: Double, maxLat: Double, maxLng: Double): List[StreetEdge] = db.withSession { implicit session =>
+  def selectAuditedStreetsIntersecting(minLat: Double, minLng: Double, maxLat: Double, maxLng: Double): List[(StreetEdge, Boolean)] = db.withSession { implicit session =>
```

Comment:
This function is used in the Neighborhoods API functions in `ProjectSIdwalkAPIController.scala`. And both the Streets and Neighborhoods APIs use the class `AccessScoreStreet`, which I added the flag to in this PR. So I decided to have the Neighborhoods API query the boolean flag, but then ignore it in later calculations so that both APIs could used the same class. If you think it's clearer, I can make a separate class for either API. This would also involve making separate `ProjectSidwalkAPIController.computeAccessScoresForStreets()` functions, but it should not be that difficult.

Reply:
Yep that's totally fine! This mattered more when there wasn't a defined case class for this metadata. It's good how you have it now :)